# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdebluetooth/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kdebluetooth-1.0_beta1-orig/kdebluetooth/irmcsync/irmcsynckonnector/clientmanager.h kdebluetooth-1.0_beta1/kdebluetooth/irmcsync/irmcsynckonnector/clientmanager.h
--- kdebluetooth-1.0_beta1-orig/kdebluetooth/irmcsync/irmcsynckonnector/clientmanager.h	2005-03-20 16:46:48.000000000 +0000
+++ kdebluetooth-1.0_beta1/kdebluetooth/irmcsync/irmcsynckonnector/clientmanager.h	2006-04-01 22:15:10.938317000 +0000
@@ -58,7 +58,7 @@
 		~ClientManager();
 
 
-		void ClientManager::setObexClient(Client*);
+		void setObexClient(Client*);
 
 		/**
 		   Returns if the worker is idel at the monment.
